src/multiversioning.zig

```diff
@@ -1819,7 +1839,7 @@ pub fn print_information(
 
     try output.print(
         "multiversioning.releases_bundled={any}\n",
-        .{multiversion.releases_bundled.const_slice()},
+        .{multiversion.releases_bundled.slice()},
```

Comment:
Maybe it makes sense to add `fn format` to release list, but I decided not to do that right away.